Modules/mathmodule.c

```diff
@@ -3098,6 +3098,41 @@ math_isfinite_impl(PyObject *module, double x)
     return PyBool_FromLong((long)isfinite(x));
 }
 
+/*[clinic input]
+math.isnormal
+
+    x: double
+    /
+
+Return True if x is normal, and False otherwise.
+[clinic start generated code]*/
+
+static PyObject *
+math_isnormal_impl(PyObject *module, double x)
+/*[clinic end generated code: output=c7b302b5b89c3541 input=fdaa00c58aa7bc17]*/
+{
+    return PyBool_FromLong((long)isnormal(x));
+}
+
+/*[clinic input]
+math.issubnormal
+
+    x: double
+    /
+
+Return True if x is subnormal, and False otherwise.
+[clinic start generated code]*/
+
+static PyObject *
+math_issubnormal_impl(PyObject *module, double x)
+/*[clinic end generated code: output=4e76ac98ddcae761 input=9a20aba7107d0d95]*/
+{
+#if defined (__STDC_VERSION__) && __STDC_VERSION__ >= 202311L
```

Comment:
```suggestion
#if defined(__STDC_VERSION__) && __STDC_VERSION__ >= 202311L
```

No hope for a `issubnormal()` on Windows?